python/examples/handler.py

```diff
@@ -21,34 +21,53 @@
 from __future__ import absolute_import
 
 import random
-import time
 
 import tornado.gen
 
 from tchannel.tornado.dispatch import TornadoDispatcher
+from tchannel.tornado.stream import InMemArgStream
+from tchannel.tornado.util import print_arg, get_arg
 
 
+@tornado.gen.coroutine
 def say_hi(request, response, opts):
-    response.write(arg3="hi")
+    arg2 = yield get_arg(request, 1)
+    arg3 = yield get_arg(request, 2)
+    response.argstreams = [
```

Comment:
Importing `InMemArgStream` and assigning a 3-element list to `argstreams` is clunky and error-prone, we'll need to make some helper methods eventually.